test/jasmine/tests/modebar_test.js

```diff
@@ -1406,6 +1445,50 @@ describe('ModeBar', function() {
             });
         });
 
+        describe('map handlers', function() {
```

Comment:
@archmoj All appearances of `mapbox` in this test should be changed to `map`, no?

Or is this specifically testing the interaction between `mapbox` buttons and `map` traces?

Reply:
@emilykl 
This seems to be fixed.
If so please resolve it.

Reply:
Resolved by [1dd8c0e](https://github.com/plotly/plotly.js/pull/7060/commits/1dd8c0e58c6ac87bef3d10198606b374bf951bbf).